server/controllers/cv-controller.js

```diff
@@ -0,0 +1,44 @@
+const client = require('../lib/github-client');
+
+console.log(client.getUserProfile());
+function fetchingData(username, res) {
+	const userProfile = client.getUserProfile(username);
+	const PullRequest = client.getUserPullRequests(username);
+	return Promise.all([userProfile, PullRequest])
+  .then((data
```

Comment:
There's no real consensus on how you _should_ format code that uses promises (they're quite a new feature of the language) but I find it helps readability if you indent `.then` and `.catch`. So something like

```js
Promise.all([p1, p2])
  .then(data => {
     //do stuff
  })
  .catch(err => {
     //handle error
  })
```

Or you could choose your own convention for formatting promise code. The important thing is to be consistent - it makes it easier to understand the code when you come back to reading it later.